src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -46,97 +47,100 @@
                 v-model="addPersonOrgFormValid"
                 v-on:submit.prevent
               >
-                <!-- Person/Org's Name -->
-                <div class="font-weight-bold" v-if="isPerson">Person's Name</div>
-                <div class="font-weight-bold" v-if="isOrg">Corporation or Firm Name</div>
-
-                <div v-if="isPerson" class="form__row three-column mt-4">
-                  <!-- NB: only staff can change Completing Party names -->
-                  <v-text-field
-                    filled
-                    class="item"
-                    label="First Name"
-                    id="person__first-name"
-                    v-model="orgPerson.officer.firstName"
-                    :rules="firstNameRules"
-                    :readonly="isCompletingParty && !isRoleStaff"
-                  />
-                  <v-text-field
-                    filled
-                    class="item"
-                    label="Middle Name"
-                    id="person__middle-name"
-                    v-model="orgPerson.officer.middleName"
-                    :rules="middleNameRules"
-                    :readonly="isCompletingParty && !isRoleStaff"
-                  />
-                  <v-text-field
-                    filled
-                    class="item"
-                    label="Last Name"
-                    id="person__last-name"
-                    v-model="orgPerson.officer.lastName"
-                    :rules="lastNameRules"
-                    :readonly="isCompletingParty && !isRoleStaff"
-                  />
-                </div>
-                <div v-if="isOrg" class="org-name-container mt-4">
-                  <v-text-field
-                    filled
-                    class="item"
-                    label="Full Legal Corporation or Firm Name"
-                    id="firm-name"
-                    v-model="orgPerson.officer.orgName"
-                    :rules="orgNameRules"
-                  />
-                </div>
+                <!-- Person Name -->
```

Comment:
Go into setting dropdown (gear icon above diffs) and hide whitespace changes to see this more neatly.